### ISSUE TITLE
Where does Template 10 fit with WTS

### ISSUE THREAD

**Issue by VagueGit** [Microsoft/WindowsTemplateStudio#409]
Our team has been working for about a year porting a business application to UWP. We're using Template 10. Why wasn't Template 10 included as a project template with WTS?

**Comment by crutkas** [comment]
We don’t consider them competing technologies and are working with the T10 team to see how we can integrate.

T10 is a framework that handles and simplifies implementation of your UWP app though ready-made templates. WTS is built on a templating system and gives you a finer degree of customization in a friendly wizard. We generate the project based on your options and try to not get in your way as the developer.  Generate only what you need and point to documentation for the rest.

At a certain point, we had to ship as well.  We used data and the community to figure out what the top frameworks/features were for an initial "minimal viable product".  We are working with the Prism team #317 in the same token.

**Comment by mrlacey** [comment]
Template10 and WTS are separate projects started by separate teams within Microsoft, that have both gone on to take community support and involvement. 
The two projects have to different purposes and goals.
Template10 is a complete framework for building UWP apps. It has a defined way of doing things and a strong opinion about some of those.
WTS exists to help developers get started building UWP apps faster. If you're new to the platform it will help provide a common structure and help you follow some popular conventions. Even if you're a highly experienced developer it aims to help by providing some standard "plumbing" code and help build a basic structure quicker.

**WTS supports multiple frameworks, so why doesn't it support Template10?**
Many reasons:
- We had to ship a v1 and that meant we couldn't include everything from day one.
- Template10 was in the process of undergoing a major refactor that looked like it might not be backward compatible. Integrating while it's still in a large state of flux would be extra work and we wouldn't want to push people towards a version which will soon be replaced by something potentially very different. (Looking at the project again now it seems [plans for version10.2](https://github.com/Windows-XAML/Template10/issues/1453) may have been shelved.)
- [Template10 still doesn't have official VS2017 compatibility](https://github.com/Windows-XAML/Template10/issues/1418) and WTS is VS2017 only.
- It totally could though. But we'd want some support from the [core members](https://github.com/Windows-XAML/Template10/graphs/contributors) of that project to help support it. (Just like with our Prism support - #304 and we did with MVVMLight support.) Before bringing in a new platform we'd want to know we have the blessing of the people behind that project and to verify that what is produced is as they'd expect it. We also want to know there was a demand for WTS supporting this framework as adding a framework to WTS multiplies the amount of testing and effort required to support every new page, feature, and option.
